mlir/test/Integration/Dialect/SparseTensor/CPU/sparse_pack_d.mlir

```diff
@@ -140,10 +140,16 @@ module {
     sparse_tensor.print %s1 : tensor<4x3x2xf32, #BatchedCSR>
     sparse_tensor.print %s2 : tensor<4x3x2xf32, #CSRDense>
 
-    // FIXME: doing this explicitly crashes runtime
-    // bufferization.dealloc_tensor %s0 : tensor<4x3x2xf32, #CCC>
-    // bufferization.dealloc_tensor %s1 : tensor<4x3x2xf32, #BatchedCSR>
-    // bufferization.dealloc_tensor %s2 : tensor<4x3x2xf32, #CSRDense>
+    %has_runtime = sparse_tensor.has_runtime_library
```

Comment:
Note: This check is no longer needed when switching to the buffer deallocation pass. When running in codegen mode, `dealloc_tensor` will turn into a no-op.

Reply:
Can you add this information in a TODO inside the test, so we remember later